content/post/2019-10-07-Image-Creation-Process.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2019-10-07T00:00:00-00:00
+description: ""
+featuredimage: ""
```

Comment:
Please minimally add a featured image here. Would also be nice if there were some images inline to help with chunking, but having one here is important so it ties into our social automation, etc.

Reply:
Working on image now.